app/backend/src/couchers/utils.py

```diff
@@ -85,8 +85,9 @@ def create_session_cookie(token, expiry):
     cookie["path"] = "/"
     # send only on requests from first-party domains
     cookie["samesite"] = "Strict"
-    # only HTTPS sites
-    cookie["secure"] = True
+    if not config["DEV"]:
+        # only set cookie on HTTPS sites in production
+        cookie["secure"] = True
```

Comment:
I tried setting it to `False` by doing `cookie["secure"] = config["DEV"]`, but that didn't seem to work, so looks like complete absence of the `Secure` key is the only way to disable it?